examples/resources/elasticstack_kibana_alerting_rule/resource-index-rule.tf

```diff
@@ -0,0 +1,35 @@
+provider "elasticstack" {
+  elasticsearch {}
+}
+
+resource "elasticstack_kibana_alerting_rule" "my_rule" {
```

Comment:
What about using more realistic names for easier understanding/following? If I'm understanding the props right, this one could be something like `DailyDocumentCountThresholdExceeded`

Reply:
Good tip, I've made that change too in b10a90b5be59f3f263201fb398431282b83c348c